pkg/processor/trigger/partitioned/v3io/partition.go

```diff
@@ -118,3 +95,39 @@ func (p *partition) Read() error {
 		}
 	}
 }
+
+func (p *partition) seek(partitionPath string) (string, error) {
+	p.Logger.DebugWith("Seeking partition",
+		"partitionPath", partitionPath,
+		"seekType", p.v3ioTrigger.seekType)
+
+	response, err := p.v3ioTrigger.container.Sync.SeekShard(&v3iohttp.SeekShardInput{
+		Path: partitionPath,
+		Type: p.v3ioTrigger.seekType,
+	})
+	if err != nil {
+		return "", errors.Wrap(err, "Failed to seek partition")
+	}
+	defer response.Release()
```

Comment:
same as below

Reply:
I guess actually this can be done immediately. I mean, `response.Output` is already populated.